pkg/apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -188,6 +188,9 @@ type PipelineSpec struct {
 	// +required
 	InputRefs []string `json:"inputRefs"`
 
+	// Labels lists labels applied to this pipeline
+	// +optional
+	Labels map[string]string `json:"labels,omitempty"`
```

Comment:
is there a syntax or character set for these labels?

Reply:
Nope, it's just like labels for `pods`, a simple `key:value` pairs